src/handlers/canonicalFieldsHandlers.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  Request,
+  Response,
+} from 'express';
```

Comment:
I guess this style allows us to see more easily in history when a new type comes in or is removed. I am surprised that the trailing comma is OK, but that will help with the same, right?